Tasks/GooglePlayRelease/package.json

```
@@ -9,13 +9,13 @@
   "author": "Jonathan Carter",
   "license": "ISC",
   "dependencies": {
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^10.17.0",
     "adbkit-apkreader": "3.1.0",
     "azure-pipelines-task-lib": "^3.0.6-preview.0",
     "glob": "^7.0.3",
-    "googleapis": "^34.0.0",
     "google-auth-library": "^2.0.0",
```

I've explained why this is needed above.

```suggestion
    "google-auth-library": "^5.6.1",
```

---

Updated to 5.6.1 👍